Source/WEB_ROOT/Inventory/js/editAsset.js

```diff
@@ -1,4 +1,34 @@
 
+// One time page load checks
+function init(assetid)
+{
+    if($j('.serialInput')){
+		checkForDup(assetid);
+		if(document.getElementById('isStudent_dcid').value == "0")
+		{
+			document.getElementById('isStudentCheckBox').checked = false;
+		}else{
+			document.getElementById('isStudentCheckBox').checked = true;
+		}
+		
+        if(assetid == -1)
+		{
+			showHideFormElements("formElement", false);
+			showHideFormElements("assignedToElement", false);
+		}
+		else
+		{
+			showHideFormElements("formElement", true);
+			showHideFormElements("assignedToElement", true);
+		}
+        toggleIsStudent();
+    }
+    else {
+        setTimeout(check, 100); // check again in a second if class serialInput was not found

```

Comment:
Just curious John, could this be removed if the javascript was loaded at the bottom rather than the top of the page? Ensuring that the DOM had already been loaded?